apps/web/src/views/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,230 @@
+  type IconButtonProps,
+  Link,
+  type LinkProps,
+  Text,
+  type TextProps,
+  useBreakpointValue,
+} from "@chakra-ui/react";
+
+import { LogoLightIcon, TezosLogoIcon } from "../../assets/icons";
+import {
+  FacebookAccountIcon,
+  GoogleAccountIcon,
+  RedditAccountIcon,
+  TwitterAccountIcon,
+} from "../../components/AccountTile/AccountTileIcon";
+import { useColor } from "../../styles/useColor";
+
+export const Welcome = () => {
+  const color = useColor();
+  const isMobile = useBreakpointValue({ base: true, lg: false });
+
+  return (
+    <Grid
+      justifyItems="center"
+      gridGap={{ base: "36px", lg: "45px" }}
+      gridTemplateAreas={{
+        base: "'header' 'main'",
+        lg: `
+     'header header header'
+     'main main main'
+     'copyright tezos-logo social'
+    `,
+      }}
+      height="100vh"
+      padding={{ base: "24px 0 0 0", lg: "24px 46px 46px 46px" }}
+      data-testid="welcome-view"
+    >
+      <Center flexDirection="column" gridArea="header">
+        <LogoLightIcon />
+        <Heading marginTop="18px" color={color("900")} size="3xl">
+          Welcome to Umami
+        </Heading>
+        <Text marginTop="6px" color={color("600")} size="lg">
+          A powerful Tezos wallet
+        </Text>
+      </Center>
+
+      <Center gridArea="main" width="full">
+        <Center
+          flexDirection="column"
+          width={{ base: "full", lg: "510px" }}
+          height={{ base: "100%", lg: "auto" }}
+          padding={{ base: "36px 12px", lg: "36px 42px" }}
+          border="1px solid"
+          borderColor={color("100")}
+          borderTopRadius="30px"
+          borderBottomRadius={{ base: 0, lg: "30px" }}
+          backgroundColor="#1A202C"
+        >
+          <Heading color={color("900")} size="lg">
+            Continue with:
+          </Heading>
+
+          <Center flexDirection="column" gap="36px" width="full" height="full">
+            <Flex
+              gap={{ base: "24px", lg: "30px" }}
+              marginTop="30px"
+              color={color("white", "black")}
+            >
+              <SocialLoginButton aria-label="google-login" icon={<GoogleAccountIcon />} />
+              <SocialLoginButton aria-label="facebook-login" icon={<FacebookAccountIcon />} />
+              <SocialLoginButton aria-label="twitter-login" icon={<TwitterAccountIcon />} />
+              <SocialLoginButton aria-label="reddit-login" icon={<RedditAccountIcon />} />
+            </Flex>
+
+            <Center gap="4px" width="full">
+              <Divider width="full" />
+              <Text color={color("500")} size="lg">
+                or
+              </Text>
+              <Divider width="full" />
+            </Center>
+
+            <Flex flexDirection="column" gap="12px" width="full">
+              <Button width="full" size="lg" variant="primary">
+                Create a new wallet
+              </Button>
+              <Button width="full" size="lg" variant="secondary">
```

Comment:
Secondary button and text color doesn't correspond to Figma mockup
<img width="594" alt="image" src="https://github.com/user-attachments/assets/86b50f82-d8be-4ab7-b2fe-eadfe9af51dc">
<img width="429" alt="image" src="https://github.com/user-attachments/assets/8eab2f2e-18a0-404f-b579-f2ed3d5cd539">